packages/onboard-react/src/components/HardwareWalletSteps/AddressListTable.tsx

```diff
@@ -0,0 +1,126 @@
+import { FC, ReactNode, useCallback, useMemo, useState } from "react";
```

Comment:
```suggestion
import { FC, useCallback, useMemo, useState } from "react";
```
Unused import.

Reply:
@creed-victor is the "build fails with warnings" setting not enabled on this build pipeline?